docker/workspace-build/Dockerfile

```diff
@@ -90,7 +90,7 @@ RUN apt-get -y update \
 COPY poetry.lock pyproject.toml docker/workspace_build_pydep_install.sh "${ANGEL_WORKSPACE_DIR}"/
 RUN cd "${ANGEL_WORKSPACE_DIR}" \
  && "${ANGEL_WORKSPACE_DIR}/workspace_build_pydep_install.sh"
-RUN python -m nltk.downloader punkt
+# RUN python -m nltk.downloader punkt
```

Comment:
During the demo with @ovenmitt , we found this to be insufficient for whatever reason.